common/src/main/java/com/microsoft/identity/common/internal/controllers/LocalMSALController.java

```diff
@@ -316,25 +318,48 @@ public AcquireTokenResult acquireTokenSilent(
         // subsequent CacheRecords represent other profiles (projections) of this principal in
         // other tenants. Those tokens will be 'sparse', meaning that their AT/RT will not be loaded
         final ICacheRecord fullCacheRecord = cacheRecords.get(0);
-
-        if (accessTokenIsNull(fullCacheRecord)
+        if (fullCacheRecord.getAccessToken().refreshOnIsActive()){
+            Logger.info(
+                    TAG,
+                    "RefreshIn is active. This will extend your token usage in the rare case servers are not available."
+            );
+        }
+        if(parameters instanceof RefreshInTokenCommandParameters) { //refresh_in second call
+            Logger.warn(
+                    TAG + methodName,
+                    "Attempting renewal of Access Token because it's refresh-expired. "
```

Comment:
can we put the relevant times in here: when is it valid until and when the refresh time is?

Reply:
Updated.